docs/tutorials/01_introduction_cloud_runtime.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<p>It optionally returns the quasiprobabilities (only if mitigation was enabled)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/97/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>I think the final version of <code>sampler</code>  will return only quasiprobabilities, but we don't have the final version yet. So this is more or less a placeholder until we know more. </p>